src/common/admin/commands/zmq_event_subscription_change_response.h

```diff
@@ -0,0 +1,162 @@
+//===================================================================================================================
+//
+// file :		zmq_event_subscription_change_response.h
+//
+// project :		tango
+//
+// author(s) :		ingvord
+//
+// Copyright (C) :      2004-2017
+//						European Synchrotron Radiation Facility
+//                      BP 220, Grenoble 38043
+//                      FRANCE
+//
+// This file is part of Tango.
+//
+// Tango is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Tango is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+// of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License along with Tango.
+// If not, see <http://www.gnu.org/licenses/>.
+//===================================================================================================================
+#ifndef TANGO_ZMQ_EVENT_SUBSCRIPTION_CHANGE_RESPONSE_H
+#define TANGO_ZMQ_EVENT_SUBSCRIPTION_CHANGE_RESPONSE_H
+
+#include <string>
+#include <vector>
+#include <tango.h>
+#include <idl/tango.h>
+#include <tango/client/DeviceData.h>
+#include <tango/server/tango_const.h>
+#include <tango/server/except.h>
+#include <memory>
+
+namespace org
+{
+namespace tango
+{
+namespace common
+{
+namespace admin
+{
+namespace commands
```

Comment:
Hello!
This part looks very Java-ish to me ;-)...
Do we really need so many nested namespaces? I would drop some of them to improve readability of the code.

Reply:
Maybe, you are right... perhaps `org::tango::common` would be enough.

P.S. once we switch to C++17 we can replace this with `org::tango::common::admin::commands`

Reply:
I would even remove the 'org' part. I don't think we need it.

Reply:
OK, I will remove `org` but leave the rest to be consistent with the folder structure. We will see how this evolve and maybe change in the future.